app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -149,32 +163,34 @@ public void onClick(View view) {
         });
 
         // Click listeners for changing activity content
-        joystickContentBtn.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (headsetIsConnected && carIsConnected) {
-                    joystickContent.setVisibility(View.VISIBLE);
-                    eegContent.setVisibility(View.GONE);
-
-                    joystickContentBtn.setVisibility(View.GONE);
-                    eegContentBtn.setVisibility(View.VISIBLE);
-
-                    if (eegActive) {
-                        eegActive = false;
-                        stopEeg();
-                        stopGyro();
-                    }
-                }
-                else {
-                    // do nothing
-                }
-            }
-        });
 
-        eegContentBtn.setOnClickListener(new View.OnClickListener() {
+      joystickContentBtn.setOnClickListener(new View.OnClickListener() {
+          @Override
+          public void onClick(View v) {
+           //   if (headsetIsConnected && carIsConnected) {
+//                    //joystickContent.setVisibility(View.VISIBLE);
+//                    eegContent.setVisibility(View.GONE);
+//
+//                    joystickContentBtn.setVisibility(View.GONE);
+//                    eegContentBtn.setVisibility(View.VISIBLE);
+//
+//                    if (eegActive) {
```

Comment:
If the content of the app UI is switched from EEG reading controls to manual controls, the eeg and gyro processes have to be turned off. This if-statement was letting the app do this. Is it represented somewhere else? If not, it cannot be commented out.